ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/API.hs

```diff
@@ -536,21 +619,21 @@ newtype ValidLedgerState l = ValidLedgerState { getValidLedgerState :: l }
 --     1. If we are passing a block by reference, we must be able to resolve it.
 --
 --     2. If we are applying rather than reapplying, we might have ledger errors.
-type Ap :: (Type -> Type) -> LedgerStateKind -> Type -> Constraint -> Type
-data Ap m l blk c where
-  ReapplyVal ::           blk -> Ap m l blk ()
-  ApplyVal   ::           blk -> Ap m l blk ( ThrowsLedgerError m l blk )
-  ReapplyRef :: RealPoint blk -> Ap m l blk ( ResolvesBlocks    m   blk )
-  ApplyRef   :: RealPoint blk -> Ap m l blk ( ResolvesBlocks    m   blk
-                                            , ThrowsLedgerError m l blk )
+type Ap :: (Type -> Type) -> (Type -> Type) -> LedgerStateKind -> Type -> Constraint -> Type
+data Ap bm m l blk c where
```

Comment:
`bm` stands for? `base monad`?

Reply:
Yes -- the `MonadBase` class uses `b` as a convention, but then it's less clear in my opinion that it is a monad